wiki/markdown/docker.md

```diff
@@ -0,0 +1,61 @@
+# Docker
+
+In Eiffel-Intelligence source code a Dockerfile is provided which helps the developer or user to build the local Eiffel-Intellegence source code changes to a Docker image.
+With the Docker image user can tryout the Eiffel-Intelligence on a Docker Host or in a Kubernetes cluster.
```

Comment:
change tryout -> try out 
or to "try-out" according to the cambridge dictionary :)

Reply:
Fixed.